scripts/template-oss/index.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  rootModule: {
+    add: {
+      'package.json': {
+        file: 'pkg.json',
+        overwrite: false,
```

Comment:
What's the custom script needed for?

Reply:
tap 16 doesn't support coverage in typescript, so this is to use c8 for coverage. it's a stop gap to ensure it works here and then can be ported back to template-oss.